lib/SyTest/Assertions.pm

```diff
@@ -0,0 +1,209 @@
+package SyTest::Assertions;
+
+use strict;
+use warnings;
+
+use Carp;
+
+use JSON;
+
+use Exporter 'import';
+our @EXPORT_OK = qw(
+   assert_ok
+   assert_eq
+
+   assert_json_object
+   assert_json_keys
+   assert_json_list
+   assert_json_nonempty_list
+   assert_json_number
+   assert_json_string
+   assert_json_nonempty_string
+   assert_json_boolean
+
+   assert_base64_unpadded
+);
+
+use Data::Dump 'pp';
+
+our %EXPORT_TAGS = (
+   all => \@EXPORT_OK,
+);
+
+=head2 assert_ok
+
+   assert_ok( $ok, $name )
+
+Fails the test if C<$ok> is false.
+
+=cut
+
+sub assert_ok
+{
+   my ( $ok, $name ) = @_;
+   $ok or
+      croak "Failed $name";
+}
+
+=head2 assert_eq
+
+   assert_eq( $got, $want, $name )
+
+Fails the test of C<$got> is not stringily equal to C<$want>.
+
+=cut
+
+sub assert_eq
+{
+   my ( $got, $want, $name ) = @_;
+
+   defined $got && defined $want && $got eq $want or
+      croak "Got ${\ pp $got }, expected ${\ pp $want } for $name";
+}
+
+=head2 assert_json_object
+
+   assert_json_object( $obj )
+
+Fails the test if C<$obj> does not represent a JSON object (i.e. is a plain
```

Comment:
The i.e. is confusing here, as it doesn't play nicely with the "not".

Does this fail the test if it *is* a plain HASH reference, or if it *isn't*?